engines/enginetest/testutils.go

```diff
@@ -27,6 +30,12 @@ func evalNilOrPanic(f func() error, a ...interface{}) {
 	nilOrpanic(f(), a...)
 }
 
+func assert(condition bool, a ...interface{}) {
+	if !condition {
+		fmtPanic(a...)
+	}
+}
+
```

Comment:
Yes and no, let's not make too much useless docs... :)

I'll take a look at the assert lib... I just wanted to keep this short... I'm less concerned when it's not exported...